webui/react/src/components/NavigationTabbar.tsx

```diff
@@ -65,6 +66,7 @@ const NavigationTabbar: React.FC = () => {
   return (
     <nav className={css.base}>
       {contextHolder}
+      {jupyterLabModalContextHolder}
```

Comment:
**question:** @julian-determined-ai @johnkim-det, wonder if there is an "ideal" spot to place these context holders? If not it would helpful to keep it consistent somewhere. I vote somewhere more on the tail end of the JSX like (but top is ok too):
```
return (
  <div className={css.base}>
    ... JSX code ...
    {contextHolder}
  </div>
);
```